api/applications/applicationRouter.js

```diff
@@ -6,7 +6,6 @@ const router = express.Router();
 const {
   cacheSignUpData,
   sendData,
-  validateStatusRequest,
```

Comment:
Do we not need this middleware anymore?

Reply:
We felt this middleware wasn't necessary since all it does is check to see if "status" coming in on req.body is one of 3 values (approved, rejected, pending). MongoDB is already performing this check and throws back an error if it doesn't receive one of these 3 values. Since we're already handling that potential error in our try/catch block we felt it was redundant to include the validateStatusRequest.